spec/rubocop/ast/block_node_spec.rb

```diff
@@ -66,9 +66,17 @@
 
     context '>= Ruby 2.7', :ruby27 do
       context 'using numbered parameters' do
-        let(:source) { 'foo { _1 }' }
+        context 'with skipped params' do
+          let(:source) { 'foo { _7 }' }
+
+          it { expect(argument_list.size).to eq(7) }
+        end
+
+        context 'with sequential params' do
+          let(:source) { 'foo { _1 + _2 }' }
 
-        it { is_expected.to be_empty }
+          it { expect(argument_list.size).to eq(2) }
```

Comment:
Could you test the names instead of just the size please?

Reply:
Done!